src/platform/forms/components/AuthorizationComponent.jsx

```diff
@@ -5,20 +5,32 @@ import { bindActionCreators } from 'redux';
 import AlertBox from '@department-of-veterans-affairs/formation/AlertBox';
 import LoadingIndicator from '@department-of-veterans-affairs/formation/LoadingIndicator';
 
-import DowntimeMessage from '../../../../platform/forms/save-in-progress/DowntimeMessage';
+import DowntimeMessage from '../save-in-progress/DowntimeMessage';
 import DowntimeNotification, {
   externalServiceStatus,
-} from '../../../../platform/monitoring/DowntimeNotification';
-import { getFormAuthorizationState } from '../../../../applications/personalization/dashboard/helpers.jsx';
+} from '../../monitoring/DowntimeNotification';
+import { getFormAuthorizationState } from '../../../applications/personalization/dashboard/helpers';
 
 import AuthorizationMessage from './AuthorizationMessage';
 
 class AuthorizationComponent extends React.Component {
+  componentDidMount() {
+    this.authorize();
+  }
+
   componentWillUpdate() {
-    if (this.props.formConfig && !this.props.profileIsLoading) {
+    this.authorize();
+  }
+
+  authorize = () => {
+    if (
+      this.props.formConfig &&
```

Comment:
Long term if all of these props come from the state, it may be better to put this behavior in a middleware.

Reply:
Thanks for the bright idea @rianfowler.